spec/unit/property/list_spec.rb

```diff
@@ -32,11 +32,11 @@
     end
 
     it "should return the passed in array values joined with the delimiter from is_to_s" do
-      expect(@property.is_to_s(["foo","bar"])).to eq("foo,bar")
+      expect(@property.is_to_s(['foo','bar'])).to eq("['foo', 'bar']")
     end
 
-    it "should be able to correctly convert ':absent' to a string" do
-      expect(@property.is_to_s(:absent)).to eq("absent")
```

Comment:
Is the list of special bare words defined somewhere? It would be easy to add exceptions for them if that's the case.